docs/tutorials/run-kinto.rst

```diff
@@ -0,0 +1,107 @@
+.. _run-kinto:
+
+Run Kinto
+#########
+
+Mozilla demo server
+===================
+
+A Kinto instance is running at https://kinto.dev.mozaws.net/v1/
+
+It should be enough to get started, but the records are flushed every night.
+
+
+Using Docker
+============
+
+If you have Docker, Kinto can be started locally with a single command:
+
+::
+
+    sudo docker run -p 8888:8888 kinto/kinto-server
+
+The server should now be running on http://localhost:8888
```

Comment:
Scratch that; after building the docs, it appears URLs are converted to links automatically by Sphinx.